.github/workflows/socrata_upload.yaml

```diff
@@ -73,8 +74,6 @@ jobs:
 
       - name: Upload to Socrata
         env:
-          AWS_IAM_ROLE_TO_ASSUME_ARN: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
-          AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
```

Comment:
I noticed that these env vars were being passed into the script but not used, so I removed them.